cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1350,6 +1350,10 @@ class SchedulerPluginFileSchema(BaseSchema):
 
     file_path = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
     timestamp_format = fields.Str(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    node_type = fields.Str(
+        metadata={"update_policy": UpdatePolicy.UNSUPPORTED}, validate=validate.OneOf(["HEAD", "COMPUTE", "ALL"])
+    )
+    log_stream_name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
```

Comment:
do we need to add a validation for allowed chars? same question applies to file_path and timestamp_format

Reply:
Yes, added for file_path and log_stream_name. I have some questions on adding it to timestamp_format, need a follow-up